api/app/Models/AssessmentStep.php

```diff
@@ -53,4 +54,9 @@ public function poolSkills(): BelongsToMany
         return $this->belongsToMany(PoolSkill::class, 'assessment_step_pool_skill')
             ->withTimestamps();
     }
+
+    public function assessmentResults(): HasMany
```

Comment:
Hmm, do we need to add this Eloquent relationship?  I can't imagine it ever being used.  I guess it can't hurt to have it.

Reply:
I thought it might be the best way to gather up results, per Figma assessment tracker mockup, it might be a good way to group things?

Reply:
After the refinement meeting, it might be best to access results through `PoolCandidate`, but might be best to leave this relation in until we know for sure whether it is of any use.

Reply:
Okey, removed the field from the assessment step type in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8302/commits/b64bdaf4f2b6ef0410fd430d070693a82d82b1f9

Reply:
You know what... could you revert b64bdaf4f2b6ef0410fd430d070693a82d82b1f9? I like the way you implemented that! It's changed my mind. It seems safe, and at least potentially useful.

Reply:
Okey, removed the removing in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8302/commits/d52fbf1b2bacc036722a7d5a8f3a3684f0aeb1f7